SecretSanta/src/SecretSanta.Domain/Services/PairingService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SecretSanta.Domain.Models;
+using SecretSanta.Domain.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SecretSanta.Domain.Services
+{
+    public class PairingService : IPairingService
+    {
+        private ApplicationDbContext DbContext { get; set; }
+        private ThreadSafeRandom Random { get; set; }
```

Comment:
These properties could be readonly.
```suggestion
        private ThreadSafeRandom Random { get; }
```